docs/content/users/basics/commands.md

```diff
@@ -550,7 +550,7 @@ Flags:
 
 * `--all`: List unofficial *and* official add-ons. (default `true`)
 * `--list`: List official add-ons. (default `true`)
-* `--verbose` or `-v`: Output verbose error information with bash `set -x` (default `false`)
+* `--verbose`, `-v`: Output verbose error information with Bash `set -x` (default `false`)
```

Comment:
I have a hard time capitalizing 'Bash' but can live with it. 

The reality is I have a hard time with 'DDEV' :)